prediction_market_agent/agents/microchain_agent/utils.py

```diff
@@ -111,3 +140,55 @@ def saved_str_to_tmpfile(s: str) -> t.Iterator[str]:
 
     # Finally remove the temporary file
     os.remove(tmp.name)
+
+
+class MechType(str, Enum):
+    PREDICTION_WITH_RESEARCH_REPORT = "prediction-with-research-report-conservative"
+    PREDICTION_ONLINE = "prediction-online"
+
+
+def mech_request(question: str, mech_type: MechType) -> MechResult:
+    private_key = MicrochainAPIKeys().bet_from_private_key.get_secret_value()
+    with saved_str_to_tmpfile(private_key) as tmpfile_path:
+        # Increase gas price to reduce chance of 'out of gas' transaction failures
+        os.environ["MECHX_LEDGER_DEFAULT_GAS_PRICE_STRATEGY"] = "gas_station"
+        response = interact(
+            prompt=question,
+            # Taken from https://github.com/valory-xyz/mech?tab=readme-ov-file#examples-of-deployed-mechs
+            agent_id=6,
+            private_key_path=tmpfile_path,
+            # To see a list of available tools, comment out the tool parameter
+            # and run the function. You will be prompted to select a tool.
+            tool=mech_type.value,
+            confirmation_type=ConfirmationType.WAIT_FOR_BOTH,
+        )
+        result = json.loads(response["result"])
```

Comment:
```suggestion
        os.environ["MECHX_LEDGER_DEFAULT_GAS_PRICE_STRATEGY"] = None
```

I don't know how costly that strategy is. Should this be changed back like this after the mech call, so it's not changed forever across the whole project?